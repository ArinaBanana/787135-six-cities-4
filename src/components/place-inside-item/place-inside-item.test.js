import React from "react";
import renderer from "react-test-renderer";
import PlaceInsideItem from "./place-inside-item.jsx";

it(`Should render Place inside item`, () => {
  const tree = renderer.create(<PlaceInsideItem item={`WI-FI`} />).toJSON();

  expect(tree).toMatchSnapshot();
});
