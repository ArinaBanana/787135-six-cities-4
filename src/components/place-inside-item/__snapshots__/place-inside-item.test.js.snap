// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Place inside item 1`] = `
<li
  className="property__inside-item"
>
  WI-FI
</li>
`;
